lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -88,7 +88,18 @@
       - The database state
     required: false
     default: present
-    choices: [ "present", "absent" ]
+    choices: [ "present", "absent", "dump", "restore" ]
+    backup:
```

Comment:
What are these three lines for?
I don't see any `backup` option, also the formatting is wrong.